### ISSUE TITLE
English

### ISSUE THREAD

**Issue by Derpolino** [winterfeel/pic2text#1]
Can you write a README in english please ?

Thanks !

**Comment by winterfeel** [comment]
@Derpolino
Of course! It's Okay but I thought I've already written a English version now.You can check the README file,I wrote English below Chinese each line.
Thanks.
